### PR TITLE
[hotfix][postgres] Remove unsupported erroneous example code from documentation

### PR DESCRIPTION
**Pull request by gtk96**
image

### CODE REVIEW

**Comment by gtk96** [comment]
@ruanhang1993 @Jiabao-Sun PTAL

File: docs/content.zh/docs/connectors/flink-sources/postgres-cdc.md [hunk 1]
```diff
@@ -470,7 +470,6 @@ public class PostgresParallelSourceExample {
                         .slotName("flink")
                         .decodingPluginName("decoderbufs") // use pgoutput for PostgreSQL 10+
                         .deserializer(deserializer)
-                        .includeSchemaChanges(true) // output the schema changes as well
```
Comment (yuxiqian) [inline docs/content.zh/docs/connectors/flink-sources/postgres-cdc.md:473]: Seems this undocumented method still exists in `PostgresSourceExampleTest` and `org.apache.flink.cdc.connectors.postgres.source.PostgresSourceBuilder#includeSchemaChanges`. Should they be removed, too?

Comment (gtk96) [reply docs/content.zh/docs/connectors/flink-sources/postgres-cdc.md:473]: > Seems this undocumented method still exists in `PostgresSourceExampleTest` and `org.apache.flink.cdc.connectors.postgres.source.PostgresSourceBuilder#includeSchemaChanges`. Should they be deprecated / removed, too?

Yes, I think it should be removed as well, thank you for reminding me

**Review by yuxiqian — APPROVED** [review]

**Comment by gtk96** [comment]
@leonardBang @ruanhang1993 PTAL

**Review by leonardBang — APPROVED** [review]
+1

**Pull request merged**